ckanext/switzerland/dcat/harvesters.py

```diff
@@ -58,6 +70,19 @@ def _get_guid(self, dataset_dict, source_url=None):  # noqa
                             'config (%s)' % (org.id, dataset_dict['owner_org'])
                         )
                         return None
+
+                    if self.config.get('excluded-dataset-identifiers'):
+                        dataset_identifier = guid.split('@')[0]
```

Comment:
Why is the identifier in the config only the part before the '@'? The identifier is only unique as a whole and maybe in the future we have a source that provides datasets for multiple organizations.